sbpy/spectroscopy/tests/test_spec_remote.py

```diff
@@ -0,0 +1,50 @@
+import astropy.units as u
+from ..spectroscopy import nophotod_prodrate
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+import numpy as np
+
+
+@remote_data
+def test_remote_prodrate_simple():
+
```

Comment:
Add a reference to https://ui.adsabs.harvard.edu/#abs/2012ApJ...756...80D/ in docstring or comment